File: Beecrowd/Beginner/1101 - 1190/1131 - Grenais/C++/1131_Grenais.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int inter = 0, gremio = 0, draw = 0, grenais = 0;

    while (true) {
        int i, g;
        cin >> i >> g;

        if (i > g) inter++;
        else if (i < g) gremio++;
        else draw++;

        grenais++;

        cout << "Novo grenal (1-sim 2-nao)" << endl;

        int option;
        cin >> option;

        if (option == 2) break;
    }

    cout << grenais << " grenais" << endl;
    cout << "Inter:" << inter << endl;
    cout << "Gremio:" << gremio << endl;
    cout << "Empates:" << draw << endl;

    if (inter > gremio) cout << "Inter venceu mais" << endl;
    else if (inter < gremio) cout << "Gremio venceu mais" << endl;
    else cout << "Nao houve vencedor" << endl;
    
    return 0;
}

File: Beecrowd/Beginner/1101 - 1190/1131 - Grenais/Python/1131_Grenais.py
inter = 0
gremio = 0
draw = 0
grenais = 0

while True:
    i, g = map(int, input().split())

    if i > g:
        inter += 1
    elif i < g:
        gremio += 1
    else:
        draw += 1

    grenais += 1

    print("Novo grenal (1-sim 2-nao)")

    option = int(input())

    if option == 2:
        break

print(f"{grenais} grenais")
print(f"Inter:{inter}")
print(f"Gremio:{gremio}")
print(f"Empates:{draw}")

if inter > gremio:
    print("Inter venceu mais")
elif inter < gremio:
    print("Gremio venceu mais")
else:
    print("Nao houve vencedor")
